src/containers/App/App.js

```diff
@@ -185,9 +182,25 @@ export function App({
 
   const showLoadingState = isPropertiesPlaceholder || isMessagesPlaceholder;
   const isFetchingConfig = isFetchingProperties || isFetchingMessages;
-  const loadingConfigError = propertiesError || messagesError;
 
-  useWebSocketReconnected(refetchProperties, webSocketConnected);
+  const {
+    data: extensions = [],
+    error: extensionsError,
+    refetch: refetchExtensions
+  } = useExtensions(
+    { namespace: tenantNamespace || ALL_NAMESPACES },
+    { enabled: !isFetchingConfig }
+  );
+
+  const loadingConfigError =
+    propertiesError || messagesError || extensionsError;
```

Comment:
no need to include extensions here as they don't block / affect the rest of the app, if we fail to load them we just won't show the nav items for them but everything else in the app works as expected.